kuma/core/tests/test_middleware.py

```diff
@@ -74,11 +74,12 @@ def test_force_anonymous_session_middleware(rf, settings):
 
 @pytest.mark.parametrize(
     'host,key,expected',
-    (('beta', 'BETA_HOST', 'kuma.urls_beta'),
-     ('beta-origin', 'BETA_ORIGIN', 'kuma.urls_beta'),
+    (('beta', 'BETA_HOST', None),
+     ('wiki', 'WIKI_HOST', None),
+     ('beta-origin', 'BETA_ORIGIN', None),
      ('demos', 'ATTACHMENT_HOST', 'kuma.urls_untrusted'),
      ('demos-origin', 'ATTACHMENT_ORIGIN', 'kuma.urls_untrusted')),
-    ids=('beta', 'beta-origin', 'attachment', 'attachment-origin')
+    ids=('beta', 'wiki', 'beta-origin', 'attachment', 'attachment-origin')
 )
 def test_restricted_endpoints_middleware(rf, settings, host, key, expected):
```

Comment:
Note-to-Ryan: This test should be exclusively about the attachments (trusted/untrusted)